config/nrfconnect/chip-module/Kconfig

```diff
@@ -135,16 +135,45 @@ config CHIP_FACTORY_DATA_MERGE_WITH_FIRMWARE
 	  As a result, a new output file merged.hex will consist of all partitions including 
 	  factory data.
 
-# Use default certificates without generating or providing them
-config CHIP_FACTORY_DATA_USE_DEFAULT_CERTS
-	bool "Use default certificates located in Matter repository"
-	default y
+# Select source of the certificates
+choice CHIP_FACTORY_DATA_CERT_SOURCE
+	prompt "Attestation certificate file source"
+	default CHIP_FACTORY_DATA_USE_DEFAULT_CERTS
+
+	config CHIP_FACTORY_DATA_USE_DEFAULT_CERTS
+		bool "Use pre-generated development certificates"
+		help
+	        Use pre-generated certificate files from the credentials/development/attestation/
```

Comment:
Something is wrong with indentation.

Reply:
Yeah, we are usually using space indentation, this setting clashed somehow with the Kconfig configuration. The normal hassle :)
Fixed this.